pom.xml

```
@@ -89,8 +89,9 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
+<!--        <maven.compiler.source>11</maven.compiler.source>-->
+<!--        <maven.compiler.target>8</maven.compiler.target>-->
```

What about to keep this?
```suggestion
        <maven.compiler.target>8</maven.compiler.target>
```